CI/travis/make_linux

```diff
@@ -37,6 +24,15 @@ handle_default() {
 		echo "### Deleting install_manifest.txt (created as root)"
 		sudo rm ./install_manifest.txt
 	fi
+	if command_exists sphinx-build ; then
```

Comment:
do we need to build docs on all deb builds or should we split the build into a `handle_doc_build()`?
the `handle_default()` rule looks complicated and gets run for all Ubuntu build-runs;
if we can simplify it a bit, would be neat;

Reply:
it depends on how many of the deb/rpms/tarballs we want with the doc, or not.

If we decide that the docs are all on-line - and they don't exist at all on the deb/rpm - that's an option.